content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: blog
+title:  The Kubernetes "Whatever-your-heart-desires" Operator
+date:   Friday, May 25th 2018
+---
+
+## Introduction
+
+Today, we are excited to announce the Kubernetes Airflow Operator; a new mechanism for launching containers using the Kubernetes cloud deployment framework. 
```

Comment:
Today, we are excited to announce the Kubernetes Airflow Operator; a mechanism for Airflow, a popular DAG orchestration framework to make native use of Kubernetes APIs.

Reply:
"workflow orchestration framework"